src/Language/PureScript/Ide/Externs.hs

```diff
@@ -95,6 +95,7 @@ convertDecl P.EDValue{..} = Just $ IdeDeclValue $
   IdeValue edValueName edValueType
 convertDecl P.EDClass{..} = Just (IdeDeclTypeClass edClassName)
 convertDecl P.EDInstance{} = Nothing
+convertDecl P.EDKind{} = Nothing
```

Comment:
Oh sry I read that as though they didn't :D the changes required are in `extractSpans` in `Ide.SourceFile` where I used an `Either` to disambiguate names on the type and value level